src/wifi.rs

```diff
@@ -1355,3 +1439,90 @@ impl WifiWait {
         }
     }
 }
+
+#[cfg(feature = "async")]
```

Comment:
As per above, you **do not have to protect** with `feature = async` anything here.